src/components/articles-components/SingleArticle.jsx

```diff
@@ -21,19 +22,21 @@ const SingleArticle = () => {
       })
       .catch((error) => {
         console.error("Error fetching article:", error);
+        setIsError("Error retreiving Article! Please try again.")
```

Comment:
when you have a variable starting with `is` it is convention for that to be a boolean